packages/panna-sdk/src/react/components/collectibles/select-recipient-step.tsx

```diff
@@ -10,15 +10,30 @@ import {
   FormMessage
 } from '../ui/form';
 import { Input } from '../ui/input';
+import { Typography } from '../ui/typography';
 import { SendCollectibleFormData } from './schema';
 
 type SelectRecipientStepProps = {
   form: UseFormReturn<SendCollectibleFormData>;
 };
 
+const ONLY_DIGITS_REGEX = /^\d+$/;
+
 export function SelectRecipientStep({ form }: SelectRecipientStepProps) {
   const { next } = useDialogStepper();
-  const { recipientAddress } = form.watch();
+  const { recipientAddress, collectible, amount } = form.watch();
+
+  const handleSubmit = async () => {
+    const isFieldValid = await form.trigger();
+    if (isFieldValid) {
+      next();
+    }
+  };
+
+  const handleMaxValue = () => {
+    const maxValue = collectible?.value ?? '1';
```

Comment:
Probably we do not need the default value here? If still necessary, it must come from the `MIN_VALUE` constant?